docs/development/contributing/ways-to-contribute.rst

```diff
@@ -0,0 +1,95 @@
+
+Ways to contribute
+==================
+
+There are four main ways you can contribute to imitation:
+
+* `Reporting bugs <#reporting-bugs>`_
```

Comment:
These hyperlinks seem broken (anchors are #id1, #id2, etc not #reporting-bugs, #suggesting-new-features in https://imitation--484.org.readthedocs.build/en/484/development/contributing/ways-to-contribute.html)